Gopkg.toml

```diff
@@ -94,3 +94,8 @@
 [[override]]
   name = "golang.org/x/sys"
   revision = "f49334f85ddcf0f08d7fb6dd7363e9e6d6b777eb"
+
+[[override]]
```

Comment:
For a library like duffle, we shouldn't use overrides, because downstream consumers won't get the pinned revision. (Overrides only apply to the current project, not someone relying on that project) 

Create a new file `pkg/dep.go` and add an empty import 

```
# Convert transitive deps to direct deps so that we can use constraints in our Gopkg.toml
import (
 _ "github.com/Azure/go-autorest/autorest"
)

Then change this override to a constraint.

Now when someone (like Porter or DockerApp) uses duffle, they will also get the right version of go-autorest. 👍

Reply:
The other option is to use a newer version of `k8s.io/client-go` which is compatible with the newer version of `github.com/Azure/go-autorest`. Of course, that's a change with larger implications. I'll apply your sugestion.

Reply:
Thanks for making the change, looks good! 👍